(DOCSP-22940): Created new Realm Web SDK Install Page and Updated (#2035)
mongodb/docs-realm: source/web/install.txt

> .. _install:

Reviewer: rename this to be `web-install`. remember the refs are global for the repo

=========================
Install the Realm Web SDK
=========================

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

Overview
--------

This page shows you how to install the Realm Web SDK to use with your application. 

Before You Begin
----------------

> Before you begin installation, you'll need to :ref:`create an {+app+}<create-a-realm-app>` for 

Reviewer: looks like we don't have the {+app+} source constant anymore. i believe nate removed last week. put in the relevant test.

this is causing the build to fail - https://github.com/mongodb/docs-realm/runs/7501851997?check_suite_focus=true

> Before you begin installation, you'll need to :ref:`create an {+app+}<create-a-realm-app>` for 

Reviewer: ```suggestion
Before you begin installation, you'll need to :ref:`create an App Services App<create-a-realm-app>` for 
```

your web app to use.

Install the Web SDK
-------------------

.. tabs::
   
   .. tab:: npm
      :tabid: npm
      
      .. code-block:: shell
         
         npm install realm-web
   
   .. tab:: yarn
      :tabid: yarn
      
      .. code-block:: shell
         
         yarn add realm-web
   
   .. tab:: CDN
      :tabid: CDN
      
      Add a ``<script>`` tag to the ``<head>`` of your HTML file to load
      the Realm Web SDK as a global variable from a content delivery
      network.
      
      Use the most recent version:

      .. code-block:: html
         
         <script src="https://unpkg.com/realm-web/dist/bundle.iife.js"></script>
      
      Or import a specific version:

      .. code-block:: html
         
         <script src="https://unpkg.com/realm-web@1.5.1/dist/bundle.iife.js"></script>

> 

Reviewer: ```suggestion

.. _web-install-node

```

Install for Node.js
-------------------

> In order to use Realm Web for Node.js environments, you must add two 
> additional peer dependendencies during installation:

Reviewer: ```suggestion
In order to use Realm Web for Node.js environments, you must
install two additional peer dependencies:
```

=========================

.. contents:: On this page
   :local: 
   :backlinks: none
   :depth: 2
   :class: singlecol

Overview
--------

This page shows you how to install the Realm Web SDK to use with your application. 

Before You Begin
----------------

Before you begin installation, you need to :ref:`create an App Services App<create-a-realm-app>` for 
your web app to use.

Install the Web SDK
-------------------

.. tabs::
   
   .. tab:: npm
      :tabid: npm
      
      .. code-block:: shell
         
         npm install realm-web
   
   .. tab:: yarn
      :tabid: yarn
      
      .. code-block:: shell
         
         yarn add realm-web
   
   .. tab:: CDN
      :tabid: CDN
      
      Add a ``<script>`` tag to the ``<head>`` of your HTML file to load
      the Realm Web SDK as a global variable from a content delivery
      network.
      
      Use the most recent version:

      .. code-block:: html
         
         <script src="https://unpkg.com/realm-web/dist/bundle.iife.js"></script>
      
      Or import a specific version:

      .. code-block:: html
         
         <script src="https://unpkg.com/realm-web@1.5.1/dist/bundle.iife.js"></script>

> .. _web-install-node

Reviewer: incorrect ref syntax
```suggestion
.. _web-install-node:
```

Author: OH that makes sense
